injector.js

```diff
@@ -165,18 +165,18 @@ Injector.prototype._applyMaps = function() {
  *     .mock('module/a', 123)
  *     .require('module/a');       // => 123
  */
-Injector.prototype.mock = function(id, value) {
+Injector.prototype.mock = function (id, value) {
 
   // Support object notation
-  if (id && typeof id === 'object') {
-    _.each(id, function(value, key) {
+  if (id && _.isObject(id)) {
```

Comment:
`id` check is not needed if we use `isObject`. That was a protection against `null`

```suggestion
  if (_.isObject(id)) {
```